src/components/cards/Card.tsx

```diff
@@ -1,21 +1,18 @@
 import React from "react";
+import { observer } from "mobx-react";
 import { Card as CardType } from "../../types";
 import "./Card.scss";
+// import { useMainStore } from "../../hooks/useMainStore";
 
 export type CardProps = {
   card: CardType;
 };
-export const Card = ({ card }: CardProps): JSX.Element => {
-  const [flipCard, setFlipCard] = React.useState(false);
+export const Card = observer(({ card }: CardProps): JSX.Element => {
+  // const { game } = useMainStore();
   const cardCSS = `front${card.face}${card.suit}`;
```

Comment:
Not the best solution but we can come back to it later

Reply:
agreed.